core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -91,34 +91,34 @@
     {% if username %}
       <li id="tutorialPreviewTab" ng-class="{'active': getTabStatuses().active === 'preview'}">
         <a href="#" tooltip="Preview" tooltip-placement="bottom" ng-click="selectPreviewTab()" class="protractor-test-preview-tab">
-          <span class="glyphicon glyphicon-play"></span>
+          <i class="material-icons">&#xE037;</i>
```

Comment:
These icons all seem a bit off-center to me. Do you feel that too? If so, I think it might be good to change padding-top: 17px to 16px instead in their CSS.